Source/DtoImporterService/LibraryBookImporter.cs

```diff
@@ -88,12 +91,29 @@ var items_lbs
 			return qtyNew;
 		}
 
+		/*
+		 * Subscription Plan Names:
+		 * 
+		 * US:	"SpecialBenefit"
+		 * IT:	"Rodizio"
+		 * 
+		 * Audible Plus Plan Names:
+		 * 
+		 * US:	"US Minerva"
+		 * IT:	"Audible-AYCL" 
+		 *
+		 */
+
 
 		//This SEEMS to work to detect plus titles which are no longer available.
 		//I have my doubts it won't yield false negatives, but I have more
 		//confidence that it won't yield many/any false positives.
 		private static bool isPlusTitleUnavailable(ImportItem item)
 			=> item.DtoItem.IsAyce is true
-			&& item.DtoItem.Plans?.Any(p => p.PlanName.ContainsInsensitive("Minerva") || p.PlanName.ContainsInsensitive("Free")) is not true;
+			&& item.DtoItem.Plans?.Any(p =>
```

Comment:
Is AYCE still used anywhere or did they scrap that altogether?

Reply:
`DtoItem.IsAyce` is still in use, but none of the "Plan" names use it as far as I'm aware. But my awareness only extends to US and Italy.